app/mailers/idea_mailer.rb

```diff
@@ -7,4 +7,11 @@ def assigned_idea_email_template(user, idea)
     set_personalisation(title: idea.title)
     mail(to: user.email)
   end
+
+  def status_change_email_template(user, idea)
+    template = 'd816f609-bc00-4f79-961c-da93ffdb8471'
+    set_template(template)
+    set_personalisation(title: idea.title, status: idea.status)
```

Comment:
Could we translate this? The translations are merged in so I think `t(idea.status)` should work here now

Reply:
Hi @davidkelliott I have added in the change to use the translated idea_status - do you want to take another look when you get a chance? Thanks!